CanvasPlusPlayground/Views/CalendarView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CalendarView.swift
+//  CanvasPlusPlayground
+//
+//  Created by Jiyoon Lee on 9/17/24.
+//
+
+import SwiftUI
+
+struct CalendarView: View {
+    let course: Course
+   @State var courseManager: CourseManager
+   init(course: Course) {
+       self.course = course
+       _courseManager = .init(initialValue: CourseManager())
+   }
+   var body: some View {
+       List(courseManager.courses, id: \.id) { course in
+           if let icsURL = course.calendar?.ics {
+               Text(icsURL)
```

Comment:
Let's start small and just download the ics file for the user to export to their Files.